src/CQRS/LeanCode.CQRS.AspNetCore/ServiceCollectionCQRSExtensions.cs

```diff
@@ -30,6 +32,28 @@ TypesCatalog handlersCatalog
 
         return new CQRSServicesBuilder(serviceCollection);
     }
+
+    public static CQRSServicesBuilder AddClientsUpdates(this IServiceCollection serviceCollection)
```

Comment:
I think we'll need more clever way of registering it. 
In essence you just need to add one assembly to the types catalog plus register some service. 

Maybe some extension method on CQRSServicesBuilder, but I need to think about it.